tests/study_tests/test_study.py

```diff
@@ -1725,3 +1725,29 @@ def test_get_metric_names() -> None:
     assert study.metric_names == ["v0"]
     study.set_metric_names(["v1"])
     assert study.metric_names == ["v1"]
+
+
+@pytest.mark.parametrize("storage_mode", STORAGE_MODES)
+def test_get_best_trial_deepcopy_control(storage_mode: str) -> None:
```

Comment:
I'm not fully sure, but we can use a simple function name here.
This is partly because this test file does not have `test_get_best_trial`, and partly because we will not add it since this test case covers full functionality of `_get_best_trial`.

```suggestion
def test_get_best_trial(storage_mode: str) -> None:
```